idseq_dag/steps/run_star.py

```diff
@@ -108,27 +108,30 @@ def __init__(self, *args, **kwargs):
         self.sequence_input_files = None
         self.validated_input_counts_file = None
 
+        insert_size_metrics_enabled = kwargs.get("insert_size_metrics_enabled", True)
```

Comment:
This became an issue during testing. In practice this probably wont happen but I want to be explicit. We never want to run this downstream so I am explicitly disabling it.